transactions/keyManager/add_key.cdc

```diff
@@ -0,0 +1,23 @@
+import KeyManager from 0xKEYMANAGERADDRESS
+
+// This transaction adds a public key to a token holder account
+// using a KeyAdder resource.
+//
+// Each token holder provides a single KeyAdder resource to the administrator
+// stored at a unique path in the administrator account.
```

Comment:
It might be worth noting that, as no API is exposed for other parties to work with these resources, storing the resources directly in the admin account's storage with well-named paths is the best pattern to use here.

Reply:
yeah I agree. Would it be better to define the paths in the contract?